wrapper/src/main/java/software/amazon/jdbc/util/SqlMethodAnalyzer.java

```diff
@@ -55,14 +58,14 @@ private String getFirstSqlStatement(final String sql) {
 
   private List<String> parseMultiStatementQueries(String query) {
     if (StringUtils.isNullOrEmpty(query)) {
-      return new ArrayList<>();
+      return singletonList("");
```

Comment:
Can you please use `Collections.singletonList("");` instead to avoid static imports?